python/README.md

```diff
@@ -0,0 +1,27 @@
+Python code for detecting faces using AWS Rekognition and Lambda
+
+train_lambda.py can be used for training the collection with images
+index_lambda.py can be used for detecting faces, checking the face i collection, sending a message if face is not present in collection.
+
+All the python code and the dependencies should be zipped into a single zip file.
+The dependencies can be installed in the /lib folder in the same folder in which you have the python code.
+The dependencies can be installed with: pip install dependency-name -t folder-path/lib/
```

Comment:
wrap commands in single ticks like this `pip install dependency-name -t folder-path/lib/`